wagtail/project_template/project_name/templates/500.html

```diff
@@ -1,8 +1,5 @@
-{% load wagtailadmin_tags i18n %}
-{% get_current_language as LANGUAGE_CODE %}
-{% get_current_language_bidi as LANGUAGE_BIDI %}
 <!DOCTYPE html>
```

Comment:
Is it worth adding a template comment to remind our future selves and those learning wagtail that they cannot use some.imports here.

Reply:
Makes sense, yep. A `{# Django template comment #}` here would get stripped out at the point of running `wagtail start`, which is probably what we want (so that it's visible to someone editing it as part of the Wagtail codebase, but doesn't affect any created projects).

Reply:
Yeah. I think that's useful. 
I can just imagine this change happening again in six months accidentally without realising we are making the same mistake

Reply:
Template comment added.